src/RequestInterface.hh

```diff
@@ -148,8 +151,8 @@ interface RequestInterface extends MessageInterface {
    *
    * @link http://tools.ietf.org/html/rfc3986#section-4.3
    * @param UriInterface $uri New request URI to use.
-   * @param bool $preserveHost Preserve the original state of the Host header.
+   * @param RequestURIOptions $options
```

Comment:
remove this, and probably the one above - doesn't give any information beyond what's in the signature

Reply:
@fredemmott resolved